src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -16,10 +18,25 @@
             + ": Displays all persons in the address book as a list with index numbers.\n"
             + "Example: " + COMMAND_WORD;
 
-
     @Override
     public CommandResult execute() {
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
-        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+        List<ReadOnlyPerson> sortPersons =  new ArrayList<ReadOnlyPerson>();
+        for (int i = 0 ; i<allPersons.size() ; i++){
+            sortPersons.add(allPersons.get(i));
+        }
+        int n = sortPersons.size();
+        for (int i = 0; i < n-1; i++)
+            for (int j = 0; j < n-i-1; j++)
+                if (sortPersons.get(j).getName().toString().toLowerCase().compareTo(sortPersons.get(j+1).getName()
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.